### ISSUE TITLE
Revision of vasoconstriction branch

### ISSUE THREAD

**Issue by BarbaraCzub** [geneontology/go-ontology#12253]
Dear Biocurators,

I am writing to request a new GO term, which arose whilst annotating paper PMID: 21768538 (Wu et al., 2011).

It is demonstrated in Figure 6 in this paper that miR-92a expression affects the regulation of vasodilation by nitric oxide.

Therefore, I wish to request the term:   ‘vasodilation by nitric oxide’

I am planning to subsequently request regulation terms via TermGenie in order to capture the effect of miR-92a on this process.

These terms would subsequently become siblings to terms such as (as well as others):
GO:0003121: regulation of vasodilation by epinephrine
GO:0003122: regulation of vasodilation by norepinephrine
GO:0003124: regulation of vasodilation by neuronal epinephrine

DbxREFs: GOC:BHF, GOC:BHF_miRNA, GOC:bc

I will look forward to hearing from you with regard to my request.

Thank you,
Barbara

cc: @RLovering 
cc: @rachhuntley

**Comment by paolaroncaglia** [comment]
Hi Barbara,

I’m afraid I have a couple of comments about this term too :-)

‘vasodilation’ is already a regulation term; it is a child of ‘regulation of blood vessel size’ and defined as “An increase in the internal diameter of blood vessels, especially arterioles or capillaries, usually resulting in a decrease in blood pressure.”. Therefore ‘vasodilation by nitric oxide’ would already capture the fact that the size of blood vessels is modified (increased = regulated) by nitric oxide. You wouldn’t need further regulation terms (and we wouldn’t want ‘regulation of regulation’ terms in the ontology). Unfortunately, I can see that the confusion may have arisen from the current existence of ‘regulation of vasodilation’ terms with several children. Some of them have a dbxref GOC:cardio so Ruth may remember something about them. I wasn’t around at the time, so I may have to check with other editors in case we need to edit the vasodilation branch - or in case I’m missing the point here…

Thanks,
Paola

**Comment by paolaroncaglia** [comment]
@RLovering

**Comment by BarbaraCzub** [comment]
Hi Paola,

Thank you for this overview.

Can I just confirm that this means that I only need to request 'vasodilation by nitric oxide', which would become is_a child term of 'vasodilation', which would be synonymous with ‘regulation of blood vessel size resulting in an increase in the internal diameter of blood vessels'?

But this also means that in order to make the annotation, which I intended, I also need to request the sibling term 'vasoconstriction by nitric oxide' (not to say 'negative regulation of vasodilation'!). However, I have just checked on Protein2GO and regulation terms for vasoconstriction are also abundant!

Thanks,
Barbara

@rachhuntley 
@RLovering

**Comment by paolaroncaglia** [comment]
Hi Barbara,

Given the confusion, I'd rather wait until I can clarify before adding terms or annotations. It seems we have a mix of terms under 'regulation of blood vessel size' - mostly old terms - my gut feeling is that we may have to edit the branch to better reflect the biology and current ontology procedures. I'll bring this up for discussion with my fellow editors and will keep you posted. Feel free to comment again if you don't hear from me within a few days.

Thanks,

Paola

**Comment by BarbaraCzub** [comment]
Thanks, Paola!

Ok, I'll will look forward to hearing some news regarding this!

Barbara

@rachhuntley 
@RLovering

**Comment by paolaroncaglia** [comment]
Note for self - 
'regulation of blood vessel size' sits directly under (is_a) 'biological regulation' -
it should be more specifically is_a 'regulation of biological quality'
and still NOT have regulation children

**Comment by BarbaraCzub** [comment]
I just looked at this paper again and 'vasoconstriction by nitric oxide' would not actually be correct, because, based on Figure 6D, nitric oxide causes vasodilation.

But silencing of KLF2 by miR-92, which happens upstream, results in a decrease in the level of vasodilation:

miR-92a  ---|  KLF2  -->  eNOS  -->  nitric oxide synthesis  -->  vasodilation 

@rachhuntley 
@RLovering

**Comment by paolaroncaglia** [comment]
Sounds like it may fit a LEGO model better than classical annotation... 
(Barbara, not sure how familiar you are with LEGO, so my comment may be addressed to Ruth and Rachael instead.)

**Comment by rachhuntley** [comment]
After discussing with Barbara, we think having synonyms of regulation of blood vessel size for vasodilation and vasoconstriction is confusing. What size is this referring to; length, width? It seems as if there should be an additional parent "regulation of blood vessel diameter".

**Comment by dosumis** [comment]
Perhaps:
'regulation of blood vessel size' -> 'regulation of blood vessel diameter' as that seems to be the intention. Synonyms on vasodilation and vasoconstriction could be changed to fit.

**Comment by paolaroncaglia** [comment]
At the editors call yesterday, we briefly went over the 'regulation of blood vessel size' branch. It was agreed that it needs a revamp. It won't be a quick fix I'm afraid. I will of course keep existing annotations into account. I'll start looking into this now and will post updates as I go. It might be more prudent for Barbara to hold onto annotating to these terms for the moment - they're confusing anyway!

**Comment by paolaroncaglia** [comment]
Notes for self: 
- 'reg. of blood vessel size' is indeed a descendant of 'reg. of biological quality', so that's ok
- 'reg. of blood vessel size' points to the OBA term 'blood vessel size', so I'll have to look into whether OBA has the more appropriate descendant of 'tube diameter' that I could use

**Comment by BarbaraCzub** [comment]
Thank you for this update, Paola.
We have left all of the uncertain annotations from this paper (PMID:21768538) on hold.

@rachhuntley 
@RLovering

**Comment by paolaroncaglia** [comment]
Let’s start at the top - regulation of blood vessel size:

[Term]
id: GO:0050880
name: regulation of blood vessel size
namespace: biological_process
def: "Any process that modulates the size of blood vessels." [GOC:ai]
is_a: GO:0003018 ! vascular process in circulatory system
is_a: GO:0035150 ! regulation of tube size
intersection_of: GO:0065007 ! biological regulation
intersection_of: regulates OBA:0000032 ! blood vessel size
relationship: part_of GO:0008015 ! blood circulation

“size” is generic here - of course a generic tube may increase its size by becoming longer as well as wider, but blood vessels specifically become wider (vasodilation) or narrower (vasoconstriction), so we need to link to a diameter term here. It appears that OBA doesn’t have a specific term yet; the one we need is

OBA:NEW blood vessel diameter
is_a OBA:0000032 blood vessel size
is_a OBA:0000171 tube diameter

@cmungall , is there an OBA TG instance that I could use to request this? Otherwise could you please point me in the right direction.

We could then rename GO ’regulation of blood vessel size’ as ‘regulation of blood vessel diameter’, change the intersection to point to the new OBA term, and go from there. (We’d also need to redefine it and place it under GO:0035296 ’regulation of tube diameter’, whose def, btw, may do with some changes - currently “Ensuring that a tube is the correct width.”)

Thanks,

Paola

**Comment by cmungall** [comment]
http://oba.termgenie.org/

**Comment by cmungall** [comment]
Actually, I'd like to try a different route for OBA, using DOSDPs

This may be much more opaque at first, but we're improving it. Go to:

https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/ontology/modules/entity_attribute.tsv

Edit the file, add a new line, use the next ID (+1 from previous line)

**Comment by paolaroncaglia** [comment]
@cmungall , I confirm I have permission to access TG OBA (but I'll try to follow your other suggested route, as recommended). However, FYI, there are ~22 unsatisfiable classes, preventing TG OBA to function.

**Comment by paolaroncaglia** [comment]
@cmungall done. Let's see how many attempts this will take :-)
Note the last line in the file was for OBA_0000110, but OBA IDs at the top of the file went as high as OBA_0000182, so I choose OBA_0000183 for my new term (this doesn't exist yet, as far as I can tell).

**Comment by paolaroncaglia** [comment]
Hi @cmungall ,
Not sure if the OBA term made it. Anyway I requested it in imports_request.obo,
+relationship: needs OBA:0000183 ! blood vessel diameter
Thanks,
Paola

**Comment by paolaroncaglia** [comment]
Hi @cmungall , any update on this please? :-)

@BarbaraCzub , FYI, I won't have a chance to work on this for a few days now, as I'll be on leave.

Thanks,
Paola

**Comment by BarbaraCzub** [comment]
Hi Paola, thanks for the update on this. 
Enjoy your leave and I'll wait to hear from you afterwards.

Thanks,
Barbara

@paolaroncaglia

**Comment by BarbaraCzub** [comment]
Hi Paola,

I was not sure, as I was following the conversation, where your term requests were supposed to make it, but I just had a look on QuickGO and the vasodilation and vasoconstriction are now defined as 'an increase/decrease in the internal diameter...' whereas the previous definitions were ‘regulation of blood vessel size resulting in an increase/decrease in the internal diameter of blood vessels' (however, there is nothing in the change log about these definitions having being modified recently...). Anyhow, does this mean that regulation terms for both vasodilation and -constriction are now valid? And/or are you planning to make further changes to these terms/definitions?

Thank you,
Barbara

@RLovering 
@rachhuntley 
@dosumis

**Comment by BarbaraCzub** [comment]
@paolaroncaglia

**Comment by paolaroncaglia** [comment]
Hi Barbara (and @cmungall ),

As far as I can see, the term OBA:0000183 ! blood vessel diameter hasn’t been committed yet; the pull request at https://github.com/obophenotype/bio-attribute-ontology/pull/11 is still open. (Barbara, Chris was busy with grant submissions and traveling to meetings lately, hence - possibly - the delay.) Ideally, I would like to start editing this branch from the top down, so I’d like to wait for the OBA term to be available and then go from there to do the necessary GO edits. 

As for definitions of vasodilation and vasoconstriction, I didn’t do any change in this branch, and I’m not aware that others did (as apparently confirmed by the change log). We editors are still of the opinion that current regulation terms for vasodilation and -constriction should not be valid. If you look at ‘regulation of blood vessel size’ and search for its children (http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0050880#term=children), you will see that both ‘vasodilation’ and ‘regulation of vasodilation’ are is_a children of the parent term - a good indicator that something is wrong. same for ‘vasoconstriction’ and its regulation. So yes, I am still planning to make changes in this ontology branch :-)

Thanks for your patience with this,

Paola

**Comment by BarbaraCzub** [comment]
Hi Paola,

Thank you for the update and for clarifying the situation. I see what you mean about the continuing need to tidy up this branch! However, I would vote for including the regulation terms in the new restructured branch. If you look at the example, which I pasted in above (miR-92a ---| KLF2 --> eNOS --> nitric oxide synthesis --> vasodilation), when annotating the silencing of KLF2 by miR-92a, it would not be wrong to add the AE causally_upstream_of negative regulation of vasodilation, but it would be incorrect in this case to say causally_upstream_of vasoconstriction. But, as you suggested, perhaps to should be annotated using LEGO?

Thanks,
Barbara

@rachhuntley 
@RLovering

**Comment by BarbaraCzub** [comment]
Or perhaps...  AE part_of negative regulation of vasodilation?

Anyway, just a thought!

Barbara

**Comment by paolaroncaglia** [comment]
Note for self - OBA term was committed a few hours ago. Next week I'll check if the term is available or if I have to add it again in imports_requests.obo.

**Comment by paolaroncaglia** [comment]
Note for self: OBA:0000183 ! blood vessel diameter doesn't seem to be available yet, so I requested it again in imports_requests.obo - just now. Will need to check after 24 hours or so.

**Comment by BarbaraCzub** [comment]
Dear Paola,

I hope you are well. Following a team discussion re: vasodilation and -constriction, we would like to emphasise that it is important to specify (within the definitions for these processes) that they involve the constriction and/or relaxation of smooth muscle cells. I.e. factors, which influence the vessel diameter, should be distinguished to make it clear that the changes in diameter do not occur e.g. due to proliferation of endothelial cells, which form the vessel. These aspects are not identified within the current vasodilation and -constriction definitions, which are 'A decrease in the diameter of blood vessels, especially arteries, usually causing an increase in blood pressure' and 'An increase in the internal diameter of blood vessels, especially arterioles or capillaries, usually resulting in a decrease in blood pressure.' respectively.

I hope you will find this helpful!

Kind Regards,
Barbara

@paolaroncaglia 
@rachhuntley 
@RLovering

**Comment by paolaroncaglia** [comment]
Hi Barbara,

Thanks for the input - yes that will be very helpful in trying to disentangle this biological node.
I will keep it in mind.

Best,

Paola

**Comment by paolaroncaglia** [comment]
Hi @cmungall 
Sorry to bug you again, but I still can't see OBA:0000183 ! blood vessel diameter... could you advise please? :-)
Thanks,
Paola

**Comment by BarbaraCzub** [comment]
Hello, I was wondering whether there has been any progress on this issue? Are there any conclusions to whether regulation terms for 'vasodilation' could be created, or is 'vasodilation' a regulation term in itself? Re: PMID:21768538, Figure 6.
Thanks,
Barbara

**Comment by paolaroncaglia** [comment]
Hi @BarbaraCzub ,
This issue wasn't forgotten, I just haven't had a chance to address it yet. Thanks for the reminder. I'll look into it again soon and will keep you posted.

**Comment by BarbaraCzub** [comment]
Thanks @paolaroncaglia

**Comment by paolaroncaglia** [comment]
Started with adding GO:0097746 regulation of blood vessel diameter
(and with fixing the def of ‘regulation of tube diameter’). More work needed.

**Comment by BarbaraCzub** [comment]
Thank you @paolaroncaglia 
So it seems to me the annotation, which I will ultimately be making, will be 'regulation of blood vessel diameter by/in response to NO', rather than 'regulation of vasodilation by NO'. And, I admit, this will be much clearer.

**Comment by paolaroncaglia** [comment]
More work: edited the ‘regulation of blood vessel size by renin-angiotensin’ node.

**Comment by paolaroncaglia** [comment]
Edited and rehoused ’vasoconstriction’ and ‘vasodilation’ - just those two terms, not their regulation ones or their children; more work needed.

**Comment by paolaroncaglia** [comment]
Hi @BarbaraCzub ,
I haven't examined the paper you wish to annotate, but if you'd like to use the new term GO:0097746 'regulation of blood vessel diameter', that's ready to use. You could add the information on NO as an extension.

**Comment by RLovering** [comment]
Hi Paola

to be clear are you suggesting AE: part_of response to NO

**Comment by paolaroncaglia** [comment]
Hi @RLovering ,
If the paper shows that the diameter of the blood vessels changes in response to NO, then yes, I'd suggest to annotate to 'regulation of blood vessel diameter' with an AE: part_of response to NO. Hope that helps.

**Comment by BarbaraCzub** [comment]
Hm... I have just re-read this paper and I am not convinced that this is how I should capture this information. 

Nitric oxide (NO) positively regulates blood vessel diamater (NAS), and there is experimental evidence in the paper in Figure 6, which shows that: 

1) miR92a negatively regulates NO biosynthesis (which I had annotated)
2) miR92a negatively regulates blood vessel diamater 
These effects are not direct, but depend on prior silencing of KLF2 by miR92a:

miR92a  --|  KLF2  -->  eNOS (NO synthase)  -->  vasodilation

So the annotations would be:
miR92a: negative regulation of blood vessel diameter
KLF2: positive regulation of blood vessel diameter
eNOS (NO synthase): positive regulation of blood vessel diameter

I will not include the AE: part_of response to NO, as the levels on NO depend on miR92a anyway.

Please let me know, if you disagree.

@paolaroncaglia I think I will wait until this branch has been tidied up as I need the positive and negative regulation terms, which atm exist as 'vasodilation' and 'vasoconstriction', for which regulation terms exist also...

![screen shot 2016-11-24 at 18 46 10](https://cloud.githubusercontent.com/assets/16541096/20608113/5847a3d8-b276-11e6-9b21-fe6b915e461b.png)

**Comment by paolaroncaglia** [comment]
Notes for self: I will approach the revamp of the vasodilation/vasoconstriction branch from the bottom up. To-do list:

- [ ] for clarity, rename ‘vasodilation by acetylcholine involved in regulation of systemic arterial blood pressure’ as ‘acetylcholine-mediated vasodilation involved in regulation of systemic arterial blood pressure’ (as agreed on an editors’ call); keep old name as exact synonym;

- [ ] similarly, rename ‘vasodilation by norepinephrine-epinephrine involved in regulation of systemic arterial blood pressure’ as ‘norepinephrine-epinephrine-mediated vasodilation involved in regulation of systemic arterial blood pressure’; keep old name as exact synonym;

- [ ] rename ‘vasodilation of artery involved in baroreceptor response to increased systemic arterial blood pressure’ as ‘artery vasodilation involved in baroreceptor response to increased systemic arterial blood pressure’; keep old name as exact synonym;

Then, looking at ‘regulation of vasodilation’ and children/descendants:

- [ ] rename ‘envenomation resulting in modulation of vasodilation in other organism’ as ‘envenomation resulting in vasodilation in other organism’ (1 annotation, correct under the new naming); keep old name as related synonym;

- [ ] rename ‘modulation of vasodilation in other organism’ as ‘vasodilation in other organism’ (no annotations); keep old name as related synonym; move it under (is_a) ‘vasodilation’ and delete link with ‘regulation of vasodilation’;

- [ ] merge ‘regulation of vasodilation by circulating epinephrine’ and ‘regulation of vasodilation by neuronal epinephrine’ (no annotations, no mappings, no inferences) into their parent ‘regulation of vasodilation by epinephrine’; make sure all resulting synonyms are related;

- [ ] rename ‘regulation of vasodilation by epinephrine’ (no annotations) as ‘epinephrine-mediated vasodilation’; keep old name as related synonym; change synonym scope of ‘regulation of vasodilation by adrenaline’ from exact to related; place it under (is_a) ‘vasodilation’ and delete link to ‘regulation of vasodilation’;

- [ ] merge ‘regulation of vasodilation by circulating norepinephrine’ and ‘regulation of vasodilation by neuronal norepinephrine’ (no annotations, no mappings, no inferences) into their parent ‘regulation of vasodilation by norepinephrine’; make sure all resulting synonyms are related;

- [ ] rename ‘regulation of vasodilation by norepinephrine’ (no annotations) as ‘norepinephrine-mediated vasodilation’; keep old name as related synonym; change synonym scope of ‘regulation of vasodilation by noradrenaline’ from exact to related; place it under (is_a) ‘vasodilation’ and delete link to ‘regulation of vasodilation’;

- [ ] the branch ‘regulation of relaxation of vascular smooth muscle’ is also problematic as ‘relaxation of vascular smooth muscle’ itself is_a ‘negative regulation of smooth muscle contraction’. Given that ‘regulation of relaxation of vascular smooth muscle’ and its pos. and neg. children have no annotations, I’d like to merge all 3 into ‘relaxation of vascular smooth muscle’ (making sure all resulting synonyms are related) and make the def. of ‘relaxation of vascular smooth muscle’ clearer by changing it to “A negative regulation of smooth muscle contraction resulting in relaxation of vascular smooth muscle. The relaxation is mediated…”;

- [ ] create GO:NEW ‘positive regulation of blood vessel diameter’ (“Any process that increases the diameter of blood vessels.”) and place ‘vasodilation’ (“An increase in the internal diameter of blood vessels…”) under it (is_a) rather than under ‘regulation of blood vessel diameter’;

- [ ] similarly, create GO:NEW ‘negative regulation of blood vessel diameter’ (“Any process that decreases the diameter of blood vessels.”) and place ‘vasoconstriction’ (“A decrease in the internal diameter of blood vessels…”) under it (is_a) rather than under ‘regulation of blood vessel diameter’;

(@BarbaraCzub, these are the new terms you need, so keep watching this space please, I will add the new GO IDs in a separate comment as soon as they’re ready.)

The list of to-do items below needs further examination:

- [ ] look at exp. annotations of ‘negative regulation of vasodilation’ and suggest new homes for them, possibly ‘negative regulation of blood vessel diameter’; then eliminate term, possibly by merging it into ‘vasodilation’;

- [ ] same for ‘positive regulation of vasodilation’;

- [ ] then look at ‘regulation of vasodilation’; it should have no children left; look at its annotations (and InterPro mappings!) and suggest new homes for them; then eliminate term, possibly by merging it into ‘vasodilation’;

- [ ] Then work on vasoconstriction.

**Comment by BarbaraCzub** [comment]
Hi @paolaroncaglia , thank you for this update. I will look forward to these new GO terms becoming available.

**Comment by paolaroncaglia** [comment]
Hi @BarbaraCzub , here are the new terms:

+id: GO:0097755
+name: positive regulation of blood vessel diameter

+id: GO:0097756
+name: negative regulation of blood vessel diameter

**Comment by BarbaraCzub** [comment]
Thanks @paolaroncaglia

**Comment by paolaroncaglia** [comment]
Following up on the strategy outlined above, I will:

- [ ] Merge ‘negative regulation of vasodilation’ into ‘negative regulation of blood vessel diameter’.
FYI, my rationale for choosing to merge into ‘negative regulation of blood vessel diameter’ rather than ‘vasoconstriction’ is the following:
‘vasoconstriction’ specifically means constriction of smooth muscle cells that line the vessels;
I looked at a couple of annotations to ‘negative regulation of vasodilation’:
1) PMID:18671723 “naofen interferes with NO production” => upstream of dilation or constriction => negative regulation of blood vessel diameter;
2) PMID:24309216 “adenosine A1 antagonist DPCPX … failed to reduce vasodilatation induced by CGS21680” => not entirely clear, so safe to say upstream of dilation or constriction => negative regulation of blood vessel diameter.
‘negative regulation of vasodilation’ had 11 experimental annotations, they are in the attached table annotations_need_checking.xlsx. If curators wish to rehouse them under more specific terms, they’ll be given a chance to do so, see below.

- [ ] Merge ‘positive regulation of vasodilation’ into ‘positive regulation of blood vessel diameter’.
Rationale for doing so is similar to the above and supported by looking at a couple of annotations (PMID:23826814 and PMID:23232926).
‘positive regulation of vasodilation’ had 55 experimental annotations, they are in the attached table annotations_need_checking.xlsx, see below;

- [ ] Merge ‘regulation of vasodilation’ into ‘regulation of blood vessel diameter’.
Again this is backed up by looking at some annotations (PMID:17916776 and PMID:12730558).
‘regulation of vasodilation’ had 7 direct experimental annotations, they are in the attached table annotations_need_checking.xlsx, see below.

- [ ] ‘regulation of vasodilation’ has 5 InterPro mappings. I’ll need to look at them and suggest changes.

- [ ] Then work on vasoconstriction :-(

- [ ] When the clean-up of this branch is complete, I’ll contact the relevant MODs to let them know of the changes and point them to the spreadsheet in case they wish to rehouse any of their annotations.

**Comment by RLovering** [comment]
Thanks Paola this sounds right

**Comment by paolaroncaglia** [comment]
[annotations_need_checking.xlsx](https://github.com/geneontology/go-ontology/files/764115/annotations_need_checking.xlsx)

**Comment by paolaroncaglia** [comment]
@asangrador @almitchell @happy-lorna @ndrawlings @EBI-Hsinyu
Following changes of GO terms related to ‘vasodilation’, I merged GO:0042312 ‘regulation of vasodilation’ into ‘regulation of blood vessel diameter’. This affects InterPro because GO:0042312 ‘regulation of vasodilation’ has 5 InterPro mappings, which I suggest to move instead to GO:0042311 ‘vasodilation’. The mappings are:
IPR000147
IPR000332
IPR000356
IPR000670
IPR001483 
Thanks in advance for looking into this.

**Comment by RLovering** [comment]
Hi Paola

so for O35607 Bmpr2 which is annotated to GO:0045906 negative regulation of vasoconstriction this should be annotated with the GO term: negative regulation of blood vessel diameter.
but this doesn't imply whether the blood vessel diameter is smaller or larger as a result. How are we supposed to indicate this. The diameter size is an important aspect of human system, and neg reg of vasoconstriction does not mean vasodilation 

Sorry for taking so long to appreciate this. I think this is why the terms were created despite the problem with negative negative.

Ruth

**Comment by paolaroncaglia** [comment]
Hi @RLovering ,
I haven't fully revised the vasoconstriction branch yet, but I'll keep your comment above in mind. (Neg reg of vasoconstriction means that the diameter does not decrease, so it's certainly not the same as neg reg of diameter ;-) perhaps you meant 'reg. of diameter', but I will look into terms and annotations and formulate a plan, I do agree that these terms are problematic but hopefully annotation procedures will become clearer to us all if we focus on what the processes are and what it is that regulates them instead. Plus, LEGO is probably the best way forward here.)

**Comment by paolaroncaglia** [comment]
@RLovering , @BarbaraCzub 

Looking at the vasoconstriction branch highlights yet more work to do. At first sight, the areas in need of revision are not only the existing regulation terms, but also the link with ‘vascular smooth muscle contraction’ (currently is_a ‘vasoconstriction’). Quickly put: vasoconstriction refers to decrease in diameter of a whole blood vessel; the vessel is made of endothelial + muscle cells; ‘vascular smooth muscle contraction’ should be part_of ‘vasoconstriction’, not is_a. (And the reciprocal ‘vasoconstriction’ has_part ‘vascular smooth muscle contraction’ should be added, too.) (Note that ‘relaxation of vascular smooth muscle’ correctly is part_of ’vasodilation’ already.) Are there cases where vascular smooth muscle cells contract without causing vasoconstriction? 
As for vasodilation, the vasoconstriction node contains many terms (though, luckily, only 57 manual experimental annotations) so revising everything is not a quick thing. I’ll have a word with David Hill. Thanks for your patience…

**Comment by paolaroncaglia** [comment]
@RLovering , @BarbaraCzub, @rachhuntley 
I discussed this ticket briefly with @ukemi. He has some thoughts about it, and agrees that it would be a really good case for LEGO. I'm handing it over to him so he can continue where I left, as I'm wrapping up my ontology tickets. Thanks.

**Comment by RLovering** [comment]
Hi David

It would be good to revise this ontology. I think the simplest revision is if we can agree that the terms 
GO:0097756 negative regulation of blood vessel diameter
Definition: Any process that decreases the diameter of blood vessels. 
GO:0097755 positive regulation of blood vessel diameter
Definition: Any process that increases the diameter of blood vessels. 
are not helpful. 
Mostly I have a problem with these terms because their definitions are not equivalent to the term names. you can negatively regulate a blood vessel diameter without decreasing the diameter. 

In addition, GO:0097756 negative regulation of blood vessel diameter has only one child:
>is_a child GO:0042310 vasoconstriction
Definition: A decrease in the diameter of blood vessels, especially arteries, due to constriction of smooth muscle cells that line the vessels, and usually causing an increase in blood pressure.
I think 20 proteins are associated with the GO:0097755 term directly by manual annotations (non-ISS).

GO:0097755 positive regulation of blood vessel diameter does have 2 child terms:
>is_a child GO:0042311                vasodilation
>is_a child GO:0045906                negative regulation of vasoconstriction
I think 45 proteins are associated with the GO:0097755 term directly by manual annotations (non-ISS). 

I would like to see these 2 terms deleted
GO:0097756 negative regulation of blood vessel diameter 
GO:0097755 positive regulation of blood vessel diameter
And then the 2 child terms vasodilation and vasoconstriction would be associated directly with the grandparent term:
 GO:0097746 regulation of blood vessel diameter
Any process that modulates the diameter of blood vessels. 

And the child term negative regulation of vasoconstriction will just stay as a child to  GO:0019229
regulation of vasoconstriction.

If you agree with this I would be happy to set up a spreadsheet asking people to check whether their annotations can be associated either with vasoconstriction, or vasodilation or just with the parent regulation of blood vessel diameter.

Thanks

Ruth

**Comment by ukemi** [comment]
Hi Ruth,

Perhaps we could just merge. This is assuming that vasoconstriction and dilation include the things that control them. I think this would be consistent with a physiologst's view.

-D

**Comment by RLovering** [comment]
HI David

I assume you mean merge 

GO:0097756 negative regulation of blood vessel diameter
with GO:0042310 vasoconstriction

GO:0097755 positive regulation of blood vessel diameter
with GO:0042311 vasodilation

This sounds good to me. But then please remove the relationship of GO:0045906 negative regulation of vasoconstriction to vasodilation, as it is just a bit too complicated to have this relation. In theory negative regulation of vasoconstriction may just leave the blood vessel at 'normal' diameter rather than increasing the size. 

I thought I would check a couple of the annotations and discovered that confusingly GO:0045908 (negative regulation of vasodilation) was already merged into GO:0097756 negative regulation of blood vessel diameter

and GO:0045909 (positive regulation of vasodilation) was merged into GO:0097755 positive regulation of blood vessel diameter

I did wonder why vasoconstriction had regulation terms and vasodilation didn't. 

Oh dear, I know you don't want regulation of regulation terms, but I think the problem is that vasocontriction and dilation are not regulation in the way that we normally think of regulation. They are caused by the extent of contraction and relaxation of smooth muscles. 

In addition we have a lot of inconsistencies in these graphs: the term GO:0060087 relaxation of vascular smooth muscle exists as a part_of child of  GO:0042311 vasodilation. 

Whereas contraction of vascular smooth muscle is_a vasoconstriction. Why do these pairs have different relationships?

In addition, I am not convinced there is any difference between relaxation of vascular smooth muscle and vasodilation; or between the vasoconstriction and contraction of vascular smooth muscle. Certainly I am not convinced that this would be possible to separate experimentally.

![screenshot 2019-01-18 at 19 54 25](https://user-images.githubusercontent.com/12080010/51409879-f270dc00-1b5a-11e9-8fe7-759549460b22.png)
![screenshot 2019-01-18 at 19 54 03](https://user-images.githubusercontent.com/12080010/51409882-f4d33600-1b5a-11e9-9fa7-2ad078fa15a8.png)

What do you think of the following:

Merge three terms: GO:0097756 negative regulation of blood vessel diameter; 
GO:0014829 vascular smooth muscle contraction and GO:0042310 vasoconstriction
**Change the relationship is_a to part_of GO:0097746 regulation of blood vessel diameter**

Merge three terms:  GO:0097755 positive regulation of blood vessel diameter; GO:0042311 vasodilation and GO:0060087 relaxation of vascular smooth muscle 
**Change the relationship is_a to part_of GO:0097746 regulation of blood vessel diameter**

The logic being that you can regulate a diameter but positive and negative regulation of a diameter doesn't make sense because you are pos or neg regulating a homeostatic process (is that the right way to say that?). I discussed this with Lucie Clapp here at UCL whose researches hypertension and the truth is that there is always a degree of muscle tension in the vasculture, it is never completely relaxed otherwise blood would not be able to circulate it would just puddle (not her words).

In addition: ideally remove the relation between  GO:0045986 negative regulation of smooth muscle contraction and GO:0060087 relaxation of vascular smooth muscle, we don't have the equivalent for 
GO:0014829 vascular smooth muscle contraction ie we don't suggest this term is_a child of negative regulation of smooth muscle relaxation.

I am not totally convinced that vasoconstriction should be merged with negative regulation of vasoconstriction. As I think we are happy with the idea of negative regulation of the extent of muscle contraction.

Thanks

Ruth

**Comment by pgaudet** [comment]
Discussed with @RLovering 
Decisions: 


- [x] Merge 'regulation of blood vessel size' and 'regulation of blood vessel diameter'
- [x] Change label: GO:0097746 regulation of blood vessel diameter -> blood vessel diameter homeostasis
- [x] Change label: GO:0002034 regulation of blood vessel diameter by renin-angiotensin ->     maintenance of blood vessel diameter homeostasis by renin-angiotensin

Obsolete: 
GO:0097756 negative regulation of blood vessel diameter -> consider: vasoconstriction
GO:0097755 positive regulation of blood vessel diameter -> consider: vasodilation

**Comment by pgaudet** [comment]
Dear all, 

The proposal has been made to obsolete 
GO:0097756 negative regulation of blood vessel diameter -> consider: vasoconstriction: 16 EXP 
GO:0097755 positive regulation of blood vessel diameter -> consider: vasodilation: 45 EXP 


Contributor   (Assigned by)
--
AgBase | 7
MGI | 2
RGD | 43
UniProt | 9

There are no mappings; these terms are not in any subsets. 

We are opening a comment period for this proposed obsoletion. Since there are many annotations to be reviewed, we’d like to proceed and obsolete this term on November 27th, 2020. Unless objections are received by November 27th, 2020, we will assume that you agree to this change. You can add comments to the ticket: https://github.com/geneontology/go-ontology/issues/12253

Thanks, Pascale

**Comment by pgaudet** [comment]
Fixed AgBase annotations.

**Comment by pgaudet** [comment]
Fixed UniProt annotations.
